app/views/events/index.html.erb

```diff
@@ -139,4 +139,4 @@
       <div style="clear: both;"></div>
   </div>
 </div>
-<%= javascript_include_tag('fullpage/past_nuansa.js') %>
+<%= javascript_include_tag('event-components/overview-fullpage.js') %>
```

Comment:
I would still like to move this line from the html file to the js file (separation of concerns), but when I do it, the page won't load with the fullpage effect. Would be great if someone could figure out why!

Reply:
Yeah same, I also tried to move it but somehow it didn't work hahaha, can be added to the todo list first lol